src/modals/send-modal/send-modal.form.tsx

```diff
@@ -31,6 +31,6 @@ export const sendModalValidationSchema: SchemaOf<SendModalFormValues> = object()
 
       return false;
     }),
-  ownAccount: object().shape({}).required(),
+  ownAccount: object().shape({}).optional(),
```

Comment:
maybe `ownAccount` filed should be required when `transferBetweenOwnAccounts === true`?  (the same what you did with `receiverPublicKeyHash`)

Reply:
Thanks, I've already tried f**king with those `yup` typings

Reply:
you could just use object literal and `@ts-ignore` in this case, to ignore types

Reply:
Okay, I thought that was strictly forbidden